desktop/npm-helper.js

```diff
@@ -136,7 +136,7 @@ const commands = {
       ELECTRON_ENABLE_LOGGING: 1,
     },
     nodePathDesktop: true,
-    shell: 'webpack --config webpack.config.visdiff.js && electron ./dist/render-visdiff.bundle.js',
+    shell: 'false',
```

Comment:
That's in the separate commit for testing -- just making sure it works with the failure case we hit